no-computer.md

```diff
@@ -58,12 +88,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+So most of the time in designing the language would be on the "systems" aspect
+of it by defining where the holes should exist on the instrument. Careful 
+physics is required to tell which holes will produce what tones. It is a mixture
+though as we know what sounds we want to make, so a combination of all the holes
+should make all these sounds. I would say 80% should be spend on the systems
+aspect. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project is pretty ambitious as it defines an entire instrument, what it
+sounds like and how it is played. 
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would not be a good idea for a project since it is not on the computer. 
+Computerizing the oboe would not produce much good because if it isn't played 
+by hands, you don't need fingerings for it. 
```

Comment:
I really like this idea, and I think there are ways to make it into a CS 111 project, if you want!

Perhaps you could translate sheet music to diagrams or other notation for oboe fingering? Most of the "language design" effort would be on intermediate representations: how would you describe oboe fingerings as a data structure that could easily be created by processing sheet music? There's also the front-end and back-end parts which are more systems-y. But we could talk about ways to handle those that allow to focus first on language design. Let me know if you're interested!